pages/astro.yml

```diff
@@ -0,0 +1,52 @@
+# Sample workflow for building and deploying an Astro site to GitHub Pages
+#
+# To get started with Astro see: https://docs.astro.build/en/getting-started/
+#
+
+name: Deploy Roadmap to Pages
+
+on:
+  push:
+    branches: [main]
+    # paths:
+    # only needed when project in subfolder
+    # - "subfolder"
+    # - ".github/workflows/astro.yml"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+env:
+  PAT: '.' # default value when not using subfolders
```

Comment:
The connotation with Personal Access Token is very strong here. I would either remove the option altogether (we don't have it for other Pages starter workflow) or find a better name.

Not too sure on a better name. `SITE_PATH`? `RELATIVE_PATH`? `BUILD_PATH`?

Reply:
if we don't have it for other starters, let's remove it!